test/tbb/test_parallel_sort.cpp

```diff
@@ -261,16 +271,28 @@ void test_psort_compare_constraints() {
 
 void test_psort_cbs_constraints() {
     using namespace test_concepts::container_based_sequence;
-    using CorrectCompare = test_concepts::compare::Correct<int>;
+    using namespace test_concepts::parallel_sort_value;
+
     static_assert(can_call_parallel_sort_with_cbs<Correct>);
     static_assert(!can_call_parallel_sort_with_cbs<NoBegin>);
     static_assert(!can_call_parallel_sort_with_cbs<NoEnd>);
     static_assert(!can_call_parallel_sort_with_cbs<ForwardIteratorCBS>);
+    static_assert(!can_call_parallel_sort_with_cbs<ConstantCBS>);
+    static_assert(!can_call_parallel_sort_with_iterator<CustomValueCBS<NonMovableValue>>);
+    static_assert(!can_call_parallel_sort_with_iterator<CustomValueCBS<NonMoveAssignableValue>>);
+    static_assert(!can_call_parallel_sort_with_iterator<CustomValueCBS<NonComparableValue>>);
+
 
+    using CorrectCompare = test_concepts::compare::Correct<int>;
+    using CompareMovable = test_concepts::compare::Correct<NonMovableValue>;
```

Comment:
It is up to you, but I would rename this using and the next one to `MovableCompare` and `MoveAssignableCompare` respectively